internal/cluster/statemachine.go

```diff
@@ -183,13 +185,57 @@ func NewClusterStateMachine(th *transitionHandler) stateswitch.StateMachine {
 		PostTransition:   th.PostRefreshCluster(statusInfoPreparingForInstallationTimeout),
 	})
 
+	sm.AddTransition(stateswitch.TransitionRule{
+		TransitionType: TransitionTypeRefreshStatus,
+		SourceStates: []stateswitch.State{
+			stateswitch.State(models.ClusterStatusInstallingPendingUserAction),
+		},
+		Condition: stateswitch.And(
```

Comment:
i don't think that you need `stateswitch.And(` here because you have only one condition, `stateswitch.Not(th.IsInstalling)` is enough.